### PR TITLE
Add scutech cms driver

### PR DESCRIPTION
**Pull request by weibangtuo**
**What this PR does / why we need it**:

Driver for Scutech Cloud Migration System(CMS)

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
NONE

**Special notes for your reviewer**:
NONE

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: contrib/drivers/scutech/cms/replication.go [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+	}
+
+	if _, err := cmsadm.Up(); err != nil {
+                return nil, err
+        }
+
+	return replica, nil
+}
+
+//delete replication
```
Comment (xing-yang) [inline contrib/drivers/scutech/cms/replication.go]: space after //

File: contrib/drivers/scutech/cms/replication.go [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+	if !isPrimary {
+		return nil
+	}
+
+	cmsadm := NewCmsAdm()
+	_, err := cmsadm.DeleteTask()
+	return err
+}
+
+//start replication
```
Comment (xing-yang) [inline contrib/drivers/scutech/cms/replication.go]: space after //

File: contrib/drivers/scutech/cms/replication.go [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+	if !isPrimary {
+		return nil
+	}
+
+	cmsadm := NewCmsAdm()
+	_, err := cmsadm.Up()
+	return err
+}
+
+//stop replication
```
Comment (xing-yang) [inline contrib/drivers/scutech/cms/replication.go]: space after //

File: contrib/drivers/scutech/cms/replication.go [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+		return nil
+	}
+
+	cmsadm := NewCmsAdm()
+
+	_, err := cmsadm.Down()
+	return err
+}
+
+//
```
Comment (xing-yang) [inline contrib/drivers/scutech/cms/replication.go]: Add comments

File: contrib/drivers/scutech/cms/replication.go [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+func (r *ReplicationDriver) FailoverReplication(opt *pb.FailoverReplicationOpts) error {
+	glog.Infoln("CMS failover ....")
+
```
Comment (xing-yang) [inline contrib/drivers/scutech/cms/replication.go]: Please add some comments explaining why this function does not do anything.  See example comments in DRBD driver:

https://github.com/opensds/opensds/blob/master/contrib/drivers/drbd/replication.go#L235

**Comment by xing-yang** [comment]
Please check CI failures.

**Comment by xxwjj** [comment]
LGTM

File: contrib/drivers/scutech/cms/replication.go [hunk 6]
```diff
@@ -0,0 +1,127 @@
+// Licensed under the Apache License, Version 2.0 (the "License");
```
Comment (leonwanghui) [inline contrib/drivers/scutech/cms/replication.go]: Please add the copyright header.

File: contrib/drivers/scutech/cms/scmstask.go [hunk 7]
```diff
@@ -0,0 +1,60 @@
+// Licensed under the Apache License, Version 2.0 (the "License");
```
Comment (leonwanghui) [inline contrib/drivers/scutech/cms/scmstask.go]: Same here

File: contrib/drivers/scutech/cms/replication.go (hunk 6 above)
Comment (xxwjj) [reply contrib/drivers/scutech/cms/replication.go]: add  # Copyright 2019 The OpenSDS Authors.

Comment (xxwjj) [reply contrib/drivers/scutech/cms/replication.go]: add  # Copyright 2019 The OpenSDS Authors.

**Comment by weibangtuo** [comment]
scutech:master -> opensds:development change to scutech:development ->  opensds:development
New PR on #890.

**Pull request closed without merging**